### ISSUE TITLE
error message when using Phase_field-mechannics /EBSD_reconstruction_grain _growth _mech

### ISSUE THREAD

**Issue by zengfy-hust** [idaholab/moose#17777]
Hello everyone 👍 
       I am a new user of moose .what I want to do is to using data from EBSD as an initial condition ,then calculate the grain boundary evolution using grain growth in phase field module .I found the close one to my intentions is Phase_field-mechannics /EBSD_reconstruction_grain _growth _mech ,if you have better choice ,please let me know .
      when I use the command line 'mpirun -n 4 /home/zengfanyi/projects/moose/modules/combined/combined-opt -i EBSD_reconstruction_grain _growth _mech.i' The issue I encountered as  follow 
![ERROR](https://user-images.githubusercontent.com/56660568/117533006-d390ff00-b01c-11eb-8d25-edff38654ef3.png)
I really don't know why ,
    Thank you so much .

**Comment by dschwen** [comment]
Yeah, looks like we a) overlooked an example when we removed `ReconVarIC`, and b) aren't doing a syntax test on that example!

Here: https://github.com/idaholab/moose/commit/c9f5e688191b44d3685fa4d7fa54ca1ab2411ed5#diff-8eb5bf72e73c00758672649b6442ac584013988400dd34595bf41c1503cc0d37 is an example on how to migrate this to the new system.
